<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Output</class>
 <widget class="QWidget" name="Output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="background.qrc">:/background/back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>410</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>1220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪瑞虎宋W</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>钩子截获信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="processInfo">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>130</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="heapView">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>130</y>
     <width>410</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>堆监控</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>文件信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>日志</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>异常信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="exceptionWindow">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>259</y>
     <width>440</width>
     <height>270</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>530</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>文件监控</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTreeView" name="fileView">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>560</y>
     <width>410</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fileAccess">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>560</y>
     <width>70</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fileShareMode">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>560</y>
     <width>100</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fileCreationDisposition">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>560</y>
     <width>150</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fileFlagsAndAttributes">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>560</y>
     <width>170</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="regeditView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>410</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>231</red>
         <green>231</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="127">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>汉仪唐美人 95W</family>
    </font>
   </property>
   <property name="text">
    <string>注册表监控</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="showLogWidget">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开日志窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showHeapWidget">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开堆窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showFileWidget">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showRegWidget">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开注册表窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showExceptionWidget">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开异常窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showNetWidget">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开网络窗口</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
